CONTRIBUTING.md

```diff
@@ -55,6 +55,8 @@ point to contribute to Logstash.
 See: https://github.com/elastic/logstash/labels/adoptme
 https://github.com/elastic/logstash/labels/low%20hanging%20fruit
 
+Using IntelliJ ?  - See a detailed getting started guide [here](https://docs.google.com/document/d/1kqunARvYMrlfTEOgMpYHig0U-ZqCcMJfhvTtGt09iZg/pub)
```

Comment:
I'd remove the extra space and dash, and just say: "Using IntelliJ? See..." 

Also add a period at the end of the sentence (after the end paren).